src/components/confirm/__snapshots__/__spec__.js.snap

```diff
@@ -0,0 +1,313 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Confirm confirmButtons when custom labels are defined returns a custom labels 1`] = `
```

Comment:
could you find the buttons and snapshot those, or even just find the buttons and test the props on them? Don't need 2 full snapshots for this.